fronts-client/src/actions/__tests__/Cards.spec.ts

```diff
@@ -104,19 +104,15 @@ const insert = async (
     [uuid, id, undefined],
     collectionCapInfo ? collectionCapInfo.cap : Infinity
   );
+
+  const stateHere: any = getState();
   await dispatch(insertCardWithCreate(
     { type: parentType, id: parentId, index },
     { type: 'REF', data: parentId },
     'collection',
-    afId => () =>
-      Promise.resolve(selectCard(selectSharedState(getState()), uuid))
+    afId => () => Promise.resolve(selectCard(stateHere, uuid))
   ) as any);
 
-  // TODO: use modal service to mock return from modal
-  // if (collectionCapInfo && collectionCapInfo.accept !== null) {
-  //   dispatch(endConfirmModal(collectionCapInfo.accept));
-  // }
-
```

Comment:
Removing this comment. This test is too complicated as it is, so any new modal test should  be in a separate file